examples/tensorflow_mnist.py

```diff
@@ -12,12 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
-#!/usr/bin/env python
+# !/usr/bin/env python
+
+import shutil
+import os
 
 import tensorflow as tf
+import keras
 import horovod.tensorflow as hvd
+import numpy as np
+
 layers = tf.contrib.layers
```

Comment:
Is it possible to use `tf.layers` instead of `tf.contrib.layers` as well?

Reply:
Changed this as well.